CHANGELOG.md

```diff
@@ -164,7 +164,11 @@ Fixed [#329](https://github.com/LLNL/sundials/issues/329) so that C++20 aggregat
 
 Fixed integer overflow in the internal SUNDIALS hashmap. This resolves
 [#409](https://github.com/LLNL/sundials/issues/409) and
-[#249](https://github.com/LLNL/sundials/issues/249)
+[#249](https://github.com/LLNL/sundials/issues/249).
+
+The `CMAKE_BUILD_TYPE` defaults to `RelWithDebInfo` mode now i.e., SUNDIALS will
+be built with optimizations enabled by default. Previously the build type was
+unset by default so no optimization or debugging flags were set.
```

Comment:
Looks like this suggestion to the top level change log didn't get applied in the last set of updates (the other files look good)
```suggestion
The `CMAKE_BUILD_TYPE` defaults to `RelWithDebInfo` mode now i.e., SUNDIALS
will be built with optimizations and debugging symbols enabled by default.
Previously the build type was unset by default so no optimization or debugging
flags were set.
```